check-broken-links.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
sweet. at some point it might be nice to drop this into the README for people working on it as an fyi that it's here too.